code/core/write_sdl_ntuple.cc

```diff
@@ -301,7 +301,7 @@ void createLowerLevelOutputBranches()
     createPixelTripletCutValueBranches();
     createQuintupletCutValueBranches();
     createPixelQuintupletCutValueBranches();
-    createTrackExtensionCutValueBranches();
+//    createTrackExtensionCutValueBranches();
```

Comment:
I guess this and some following commented out code will be dropped in another PR? Or do we want to drop it here?

Reply:
I guess this will be part of Philip's upcoming clean-up PR, so let's leave it as is for now, since it doesn't break anything.